app/Models/User.php

```diff
@@ -439,7 +439,7 @@ public function getPointsWeightedAttribute(): int
 
     public function hasVerifiedEmail(): bool
     {
-        return (int) $this->getAttribute('Permissions') >= Permissions::Registered;
+        return $this->email_verified_at?->isPast() ?? false;
```

Comment:
FYI, There are ~40 users in the database with `Permissions == Registered` and a null `email_verified_at`.

The most recently used ones appear to be the result of being manually set to Registered by a moderator. Since they're going to be seeing the "please verify your email" banner, it seems like it's a non-issue.